From: Thomas Haller <thaller@redhat.com>
Date: Mon, 29 Aug 2016 17:14:04 +0200
Subject: device: fix spelling in logging

(cherry picked from commit d51f2c2a4e99799739e2adbeaf578144b556c4b9)
(cherry picked from commit b1f5d3d798498c53fe65257490b2df3e3f71e364)
---
 src/devices/nm-device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/devices/nm-device.c b/src/devices/nm-device.c
index 199acc6..305a1bb 100644
--- a/src/devices/nm-device.c
+++ b/src/devices/nm-device.c
@@ -11820,7 +11820,7 @@ _hw_addr_set (NMDevice *self,
 			       operation, addr, detail);
 		} else {
 			_LOGW (LOGD_DEVICE,
-			       "set-hw-addr: new MAC address %s not successfully set to %s (%s)",
+			       "set-hw-addr: new MAC address %s not successfully %s (%s)",
 			       addr, operation, detail);
 			success = FALSE;
 		}
